aspnetcore/mvc/controllers/testing/sample/TestingControllersSample/src/TestingControllersSample/Views/Home/Index.cshtml

```diff
@@ -27,12 +27,10 @@
                 Add New Session
             </div>
             <div class="panel-body">
-                @Html.ValidationSummary()
                 <form method="post" asp-controller="Home" asp-action="Index">
                     <fieldset class="form-group">
                         <label for="sessionName">New Session Name</label>
-                        <input type="text" class="form-control" id="sessionName" name="SessionName"
-                               placeholder="New Brainstorming Session Name"/>
+                        <input type="text" class="form-control" id="sessionName" name="SessionName" placeholder="Name" required>
```

Comment:
Let's be consistent with `input` tags in this file. For example, the button uses a self-closing tag, but this one does not.

Reply:
I missed that one below. I'm aware it's optional; I'll add it back. I just don't close them in my own stuff.